webapp/src/immersivereader.tsx

```diff
@@ -46,29 +62,98 @@ function beautifyText(content: string): string {
         );
     }
 
-    // Remove ** around bold text, replace with "
-    function cleanBold(content: string): string {
+    // If the ```block ``` annotation is used to display a block in
+    // a modal, remove it
+    function cleanHintText(content: string): string {
         return content.replace(
-            /\*\*([^*]+)\*\*/gu,
-            (matched, word, offset, s) => lf("\"{0}\"", word)
+            /^```block$[^`]*^```$/gum,
```

Comment:
There's a laundry list of these over in https://github.com/microsoft/pxt/blob/ec20d5991e3c23707a6bd5c85cd3a2a913d49a29/pxtlib/tutorial.ts#L53 -- might be able to unify that a bit? but probably good for now!

Reply:
oh man that's great